### PR TITLE
Backup filename changed

### PR DESCRIPTION
**Pull request by robindirksen1**
Changed the backup filename for running more backup each day - for example every 4 hours

### CODE REVIEW

**Comment by robindirksen1** [comment]
Also for the end user it's more clear to see when the backup is created

**Comment by anton-reutov** [comment]
I will ckeck it tommorow
Thank you

**Comment by madeITBelgium** [comment]
I think the restore script needs also a change.
https://github.com/serghey-rodin/vesta/blob/master/bin/v-restore-user#L41

**Comment by robindirksen1** [comment]
tested on centos 7 - centos-release-7-3.1611.el7.centos.x86_64

**Comment by Skamasle** [comment]
In debian 8 works fine backup rotation, I will test FTP and SFPT options

**Comment by anton-reutov** [comment]
@Skamasle, local mode also working fine ?
What about restore backups?

**Comment by Skamasle** [comment]
For now I only test local mode, and rotation working, restore working fine also in test mode

:/backup# ls -1
admin.2017-10-05_03-28-38.tar
admin.2017-10-05_03-29-02.tar
admin.2017-10-05_16-51-13.tar

v-restore-user detect new backup format fine

```
# v-restore-user admin admin.2017-10-05_16-53-27.tar

-- WEB --
2017-10-05 16:57:31 vesta1.skamasle.com

```
I test it in FTP and SFTP in this weekend

**Comment by madeITBelgium** [comment]
Restoring an old backup will not work I think.

**Comment by anton-reutov** [comment]
@madeITBelgium , are you tested it?

**Comment by robindirksen1** [comment]
@madeITBelgium @anton-reutov I changed the permission check, hope this works with the old backup notation. 

@Skamasle can you test this?

**Comment by Skamasle** [comment]
@madeITBelgium I can confirm this not works, also with your last changes

If the "if" not print noting you get an unexpected error.

`1: line 4: syntax error near unexpected token `else'
1: line 4: `    else'`

In any way you forget add [[ $2 =~ ^$1.[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9].tar$ ]]

So you do doble check to [[ $2 =~ ^$1.[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9]_[0-9][0-9]-[0-9][0-9]-[0-9][0-9].tar$ ]]

**Comment by robindirksen1** [comment]
So the way it's now should work, right?

File: bin/v-restore-user [hunk 1]
```diff
@@ -38,8 +38,16 @@ source $VESTA/conf/vesta.conf
 
 # Check backup ownership function
 is_backup_available() {
-    if ! [[ $2 =~ ^$1.[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9].tar$ ]]; then
-         check_result $E_FORBIDEN "permission denied"
+    passed=false
+    if [[ $2 =~ ^$1.[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9]_[0-9][0-9]-[0-9][0-9]-[0-9][0-9].tar$ ]]; then
+        passed=true
+    elif [[ $2 =~ ^$1.[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9]_[0-9][0-9]-[0-9][0-9]-[0-9][0-9].tar$ ]]; then
```
Comment (madeITBelgium) [inline bin/v-restore-user]: The regex is on both lines the same. Change one of the lines to the old regex:
^$1.[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9].tar$

**Comment by Skamasle** [comment]
Now working fine, but i think this code need to be improved

`    if [ $passed = false ]; then
        echo "not permissions";
       #check_result $E_FORBIDEN "permission denied"
     fi
`
This generate dual errors, when backup not exists, you get a permisions denied

 v-restore-user admin admin.2017-10-09-0.tar
not permissions
Error: backup admin.2017-10-09-0.tar doesn't exist

So you may try with this one to use vesta checkresult function:

```
is_backup_available() {
    passed=false
    if [[ $2 =~ ^$1.[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9]_[0-9][0-9]-[0-9][0-9]-[0-9][0-9].tar$ ]]; then
        passed=$?
    elif [[ $2 =~ ^$1.[0-9][0-9][0-9][0-9]-[0-9][0-9]-[0-9][0-9].tar$ ]]; then
        passed=$=
    fi
     return $?
     check_result $E_FORBIDEN "permission denied"
}
```

**Comment by robindirksen1** [comment]
So @madeITBelgium this is correct?

**Comment by anton-reutov** [comment]
i'm checking now and everything works fine so far

**Comment by dpeca** [comment]
I will put this on production machine, in next few days we will know if it needs corrections.

**Comment by dpeca** [comment]
Works fine with FTPS remote backup too.

**Pull request merged**